prow/github/fakegithub/fakegithub.go

```diff
@@ -373,6 +373,29 @@ func (f *FakeClient) AddLabel(owner, repo string, number int, label string) erro
 	return fmt.Errorf("cannot add %v to %s/%s/#%d", label, owner, repo, number)
 }
 
+// AddLabels adds a list of labels
+func (f *FakeClient) AddLabels(owner, repo string, number int, labels ...string) error {
```

Comment:
Please deduplicate code shared with the `AddLabel` function above. (i.e. make one of these functions call the other like you did for the real client functions.)